python/extensions/before_main_llm_call/_10_log_for_stream.py

```
@@ -19,8 +19,8 @@ async def execute(self, loop_data: LoopData = LoopData(), text: str = "", **kwar
                 )
             )
 
-def build_heading(agent, text: str):
-    return f"icon://network_intelligence {agent.agent_name}: {text}"
+def build_heading(agent, text: str, icon: str = "network_intelligence"):
```

Ruff (0.14.10)

Missing return type annotation for private function `build_heading`

Add return type annotation: `str`

(ANN202)